app/src/App.tsx

```diff
@@ -97,6 +102,16 @@ function AppToolbar() {
   );
 }
 
+function ScrollToTop() {
+  const { pathname } = useLocation();
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [pathname]);
+
+  return null;
+}
```

Comment:
This feels like it should be a hook, not a component

Reply:
discussed offline